Find_first_missing_positive_integer.rb

```diff
@@ -0,0 +1,33 @@
+=begin 
+This problem was asked by Stripe.
+
+Given an array of integers, find the first missing positive integer in linear time and constant space. 
+In other words, find the lowest positive integer that does not exist in the array. 
+The array can contain duplicates and negative numbers as well.
+
+For example, the input [3, 4, -1, 1] should give 2. The input [1, 2, 0] should give 3.
+
+You can modify the input array in-place.
+
+=end 
+
+def findfirstmissingpostiveint(array)
+    array = array.sort 
+    expected  = array[0]
+    (0..array.length).each do |i|

```

Comment:
can also just use a array.each loop here

Reply:
when i use that array.each for this question ([1, 2, 0]) i get [0, 1, 2] instead of 3. but for the rest im getting the value

Reply:
Please Dont consider this PR because i have given a better code in another PR ( Find firstmissingpositiveinteger #100)